### DISCUSSION TITLE
Lesson 13:  Timestamp 14:15:26 issue with 'yarn start'

### DISCUSSION THREAD

**Discussion by ChristianG77** [smartcontractkit/full-blockchain-solidity-course-py#437] [Q&A] [answered]
Hi,

I'm getting this issue with 'yarn start' with an error 'warning package.json:  No license field'.

Any help would be much appreciated.

Thanks.

![image](https://user-images.githubusercontent.com/91568178/143342366-b675bdd5-0917-4a0d-8c08-3f813b5c0fc9.png)

**Comment by cromewar** [comment, marked as answer]
Hello @ChristianG77 is because you are not on the correct folder.
type `cd front-end` or the name of the folder you have and try again.
Best regards, Cromewar.

**Reply by ChristianG77** [reply]
Thanks @cromewar .  Seems to be still have an issue once in the correct folder.

![image](https://user-images.githubusercontent.com/91568178/143366440-3223e83d-eff8-43d8-ad3d-31ce26eaec4b.png)

**Reply by cromewar** [reply]
you should run just `yarn` to install the dependencies first, al uso add the packages with yarn add, but on the frontend-folder.

**Reply by ChristianG77** [reply]
Sorry, I didn’t understand everything in your last message.

What I’ve done are the following steps as per the instructions in lesson 13:  npx –version, npm install –global yarn, yarn –version, npx create-react-app front_end –template typescript, cd front_end, yarn, yarn start

What should I do differently?

It seems to keep throwing on this error:

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/tokenize' is not defined by "exports" in /home/christian77/projects/A011_L13_defi_stake_yield_brownie/front_end/node_modules/postcss-safe-parser/node_modules/postcss/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:683:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/christian77/projects/A011_L13_defi_stake_yield_brownie/front_end/node_modules/postcss-safe-parser/lib/safe-parser.js:1:17) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

Node.js v17.1.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

![image](https://user-images.githubusercontent.com/91568178/143407742-ab9efe80-676a-438e-9a0b-b8792a9c55fa.png)
![image](https://user-images.githubusercontent.com/91568178/143407763-ce21bcf7-5db1-4ed8-b31e-9e9aafce9ce3.png)
![image](https://user-images.githubusercontent.com/91568178/143407788-dac29f37-2997-46a3-83f5-96596c57d760.png)
![image](https://user-images.githubusercontent.com/91568178/143407803-aee3b06a-bf22-486c-9165-0c57c7540528.png)
![image](https://user-images.githubusercontent.com/91568178/143407818-1699b625-ddf3-4294-8c11-a25ef756e211.png)
![image](https://user-images.githubusercontent.com/91568178/143407843-6a71fa1a-db63-4b74-9128-965f9fc550f3.png)
![image](https://user-images.githubusercontent.com/91568178/143407883-50d33639-de3f-44ba-ae5b-c7b6a299d4b5.png)

**Reply by cromewar** [reply]
It seems to be a problem with your npm version, I found the same error [here](https://stackoverflow.com/questions/69693907/error-err-package-path-not-exported-package-subpath-lib-tokenize-is-not-d), please refer to the link and see if the solution provided works for you.

**Reply by ChristianG77** [reply]
Thanks for that.

I downgraded Node.js to 16.13.0 LTS and received the following error:

![image](https://user-images.githubusercontent.com/91568178/143551187-5a646ace-ee9d-4cad-b1fa-fa364393e8cc.png)

I got it to work by changing the package.json file as per below but some of the comments said this wasn't a good idea.  Is this correct?

"start": "react-scripts start"

to

"start": "react-scripts --openssl-legacy-provider start"

**Reply by cromewar** [reply]
is not recommended as openssl legacy lacks on compatibility on some libraries or uses outdated packages, however if you were able to run it is ok, I suggest you add npm packages to ~/.bashrc on your wsl, this would help a lot with this.

**Reply by ChristianG77** [reply]
That's great feedback thanks!

So how do I 'add npm packages to ~/.bashrc'?

What would I type in the terminal?

**Reply by cromewar** [reply]
Hi, please use this as reference: https://stackoverflow.com/questions/18088372/how-to-npm-install-global-not-as-root
